exo/helpers.py

```diff
@@ -54,6 +84,11 @@ def print_yellow_exo():
     reset = "\033[0m"    # ANSI escape code to reset color
     print(f"{yellow}{exo_text}{reset}")
 
+def print_system_info():
+    system_info = get_system_info()
+    print(f"Detected system: {system_info}")
+    print(f"Using inference engine: {get_inference_engine().__class__.__name__}")
```

Comment:
This is creating a new inference engine which is quite a heavy operation / should only be done once in the app.